.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,3 +4,7 @@
 
 
 By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
+
+# Check any applicable:
+- [ ] Were any files moved? Moving files changes their URL, which breaks all hyperlinks to the files.
```

Comment:
I am not sure what this PR does. Are we trying to add a list of `types of changes` that require a PR to the PR Template?

Reply:
Resolved from offline conversation: This check is to identify PRs that may break URLs/hyperlinks to files.